src/mfa-login.ts

```diff
@@ -14,6 +14,14 @@ export interface ProfileConfiguration {
   mfaSessionValid?: boolean;
 }
 
+export interface AssumeRoleProfileConfiguration {
+  profileName: string;
+  parentProfileName: string;
+  roleArn: string;
```

Comment:
These last three should be prefixed with `aws` to be consistent with the profile configuration.